ynh_add_swap/ynh_add_swap

```diff
@@ -63,3 +71,17 @@ ynh_del_swap () {
 		rm /swap_$app
 	fi
 }
+
+# Check if the device of the main mountpoint "/" is an SD card
+#
+# return 1 if it's an SD card, else 0
```

Comment:
You usually return 1 to say no, as it's a function. It returns 0 for a success, non 0 for errors.
So here, as it's "Is it a sd card", you would say 0 to yes, success.